mod_app/models/support_models.py

```diff
@@ -31,7 +31,7 @@ def __str__(self):
         blank=False,
     )
     film = models.ForeignKey(
-        "Film", on_delete=models.CASCADE, related_name="%(class)ss", null=True
+        "Film", on_delete=models.SET_NULL, related_name="%(class)ss", null=True
```

Comment:
The cascade here means that the link model would be deleted if the film it points to gets deleted. It wouldn't explain why one of the films disappeared.

Reply:
hmm, I was worried that might be the case. I think it's still ok to go ahead with this change since we don't want to accidentally remove one of the items if it's referenced in more than one place. I couldn't see any other cascades though so I'm not sure what happened with the missing film

Reply:
User error? Or did they add it to staging by mistake?

Reply:
Likely user error but I'm still trying to see if I can figure it out